Issues with Omada v5.12

Hi,

I just updated to Omada v5.12 (officially released), and now coredns_omada seems to have issues? Or at least, I don't see to have local name resolution.

Is there a way to check for this, help debug?

Thanks!

Ok, updated `go.mod`, but it's complaining about version,
```
CGO_ENABLED=0  go build -v -ldflags="-s -w -X github.com/coredns/coredns/coremain.GitCommit=ae2bbc29-dirty" -o coredns
core/plugin/zplugin.go:59:2: /opt/coredns_omada@v1.4.2: missing go.sum entry for go.mod file; to add it:
        go mod download /opt/coredns_omada
```
But then,
```
$ go mod download /opt/coredns_omada
go: malformed module path "/opt/coredns_omada": empty path element
```
That path really is correct 😆 

Thanks!

--- a/go.mod
+++ b/go.mod
@@ -5,7 +5,7 @@ go 1.21
 require (
 	github.com/coredns/caddy v1.1.1
 	github.com/coredns/coredns v1.10.0
-	github.com/dougbw/go-omada v0.4.0
+	github.com/dougbw/go-omada v0.4.2
 	github.com/go-playground/validator/v10 v10.14.1
 	github.com/miekg/dns v1.1.50
 )

